pkgs/development/compilers/openjdk/openjfx/11.nix

```diff
@@ -83,7 +91,26 @@ in makePackage {
     substituteInPlace build.gradle \
       --replace 'mavenCentral()' 'mavenLocal(); maven { url uri("${deps}") }' \
       --replace 'name: SWT_FILE_NAME' "files('$swtJar')"
-  '';
+  '' +
+  (if withWebKit then ''
```

Comment:
Instead of `if cond then "..." else ""` you could use `lib.optionalString cond "..."`